src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,17 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService newUserSerice = new UserService();
+        User user1 = newUserSerice.findByEmail(email);
+        if (user1 == null) {
+            return false;
+        } else {
+            String correctPassword = user1.getPassword();
+            if (password.equals(correctPassword)) {
+                return true;
+            } else {
+                return false;
+            }
```

Comment:
Avoid nesting conditional operators. One `if` block is enough, think about how to combine conditions and simplify your solution.